<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JabberCat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TagsInput" name="magic_bar">
      <property name="placeholderText">
       <string notr="true" extracomment="Unused, this is set explicitly in the code to embed the shortcut.">Search…</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="roster_view">
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QListView" name="conversations_view">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QStackedWidget" name="conversation_pages">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJabberCat">
    <property name="title">
     <string>&amp;JabberCat</string>
    </property>
    <addaction name="action_manage_accounts"/>
    <addaction name="action_muc_join"/>
   </widget>
   <widget class="QMenu" name="menuContacts">
    <property name="title">
     <string>&amp;Contacts</string>
    </property>
    <addaction name="action_add_contact"/>
    <addaction name="action_invite_contact"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="action_about_qt"/>
    <addaction name="separator"/>
    <addaction name="action_open_python_console"/>
   </widget>
   <addaction name="menuJabberCat"/>
   <addaction name="menuContacts"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_manage_accounts">
   <property name="icon">
    <iconset theme="document-properties">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure accounts …</string>
   </property>
   <property name="toolTip">
    <string>Add and remove XMPP accounts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+A</string>
   </property>
  </action>
  <action name="action_muc_join">
   <property name="text">
    <string>&amp;Join Multi-User Chat…</string>
   </property>
  </action>
  <action name="action_add_contact">
   <property name="icon">
    <iconset theme="contact-new">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Add contact…</string>
   </property>
  </action>
  <action name="action_invite_contact">
   <property name="text">
    <string>&amp;Invite contact…</string>
   </property>
  </action>
  <action name="action_focus_search_bar">
   <property name="text">
    <string>Focus Search Bar</string>
   </property>
   <property name="toolTip">
    <string>Put focus on the search bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_manage_tags">
   <property name="text">
    <string>Manage tags…</string>
   </property>
   <property name="toolTip">
    <string>Add/remove tags to the selected contacts.</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="text">
    <string>Rename…</string>
   </property>
   <property name="toolTip">
    <string>Rename the contact</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_remove_contact">
   <property name="text">
    <string>Remove contact…</string>
   </property>
   <property name="toolTip">
    <string>Remove the contact from the list.</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="action_subscribe">
   <property name="text">
    <string>Subscribe to online status</string>
   </property>
   <property name="toolTip">
    <string>Ask the contact to subscribe to their online status.</string>
   </property>
  </action>
  <action name="action_unsubscribe_peer">
   <property name="text">
    <string>Unshare your online status</string>
   </property>
   <property name="toolTip">
    <string>Disallow the contact to see your online status, avatar and other public information.</string>
   </property>
  </action>
  <action name="action_subscribe_peer">
   <property name="text">
    <string>Share your online status</string>
   </property>
   <property name="toolTip">
    <string>Allows the contacts to see your online status, avatar and other public information.</string>
   </property>
  </action>
  <action name="action_add_contact_2">
   <property name="text">
    <string>Add contact…</string>
   </property>
   <property name="toolTip">
    <string>Add a new contact by entering their Jabber Address.</string>
   </property>
  </action>
  <action name="action_invite_contact_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Invite contact…</string>
   </property>
   <property name="toolTip">
    <string>Invite a contact by generating an invitation link.</string>
   </property>
  </action>
  <action name="action_open_python_console">
   <property name="text">
    <string>Open Python console…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P, Ctrl+Y, Ctrl+C</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt …</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagsInput</class>
   <extends>QLineEdit</extends>
   <header>jabbercat.widgets.tagsinput</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
